.github/workflows/express-build.yml

```diff
@@ -0,0 +1,110 @@
+        os: [ubuntu-latest, windows-latest]
+        experimental: [false]
+        node-mirror: ["https://nodejs.org/dist/"]
+        include:
+          - node-version: 13
+            os: ubuntu-latest
+            node-mirror: "https://nodejs.org/download/nightly"
+            experimental: true
+          - node-version: 13
+            os: windows-latest
+            node-mirror: "https://nodejs.org/download/nightly"
+            experimental: true
+            
+    steps:
+    - uses: actions/checkout@v2
+      
+    - name: Set up node using nvm
+      uses: dcodeIO/setup-node-nvm@v4.0.0
+      with:
+        node-version: ${{ matrix.node-version }}
+        node-mirror: ${{ matrix.node-mirror }}
+      
+    # Configure npm
+    # Skip updating shrinkwrap / lock
+    - run: npm config set shrinkwrap false
+      
+    # Remove all non-test dependencies
+    # Remove example dependencies
+    - run: npm rm --silent --save-dev connect-redis
+    
+    - name: Get Major Version Number of NodeJs Using Powershell
+      run: |
+        echo ${{ matrix.node-version }}
+        $mynodeVersion = "${{ matrix.node-version }}"
+        $mynodeMajorVersion = $mynodeVersion.split(".")[0]
+        echo "::set-env name=NodeMajorVersion::$mynodeMajorVersion"
+      shell: pwsh
+      
+      # Setup Node.js version-specific dependencies
+        
+      # mocha for testing
+      # - use 3.x for Node.js < 4
+      # - use 5.x for Node.js < 6
+      # - use 6.x for Node.js < 8
+    - name: Setup Node.js version-specific dependencies - mocha for testing - use 3.5.3 for Node.js < 4
+      run: npm install --silent --save-dev mocha@3.5.3
+      if: env.NodeMajorVersion < 4
+      
+    - name: Setup Node.js version-specific dependencies - mocha for testing - use 5.2.0 for Node.js < 6
+      run: npm install --silent --save-dev mocha@5.2.0
+      if: env.NodeMajorVersion < 6 && env.NodeMajorVersion >= 4
+        
+    - name: Setup Node.js version-specific dependencies - mocha for testing - use 6.2.2 for Node.js < 8
+      run: npm install --silent --save-dev mocha@6.2.2
+      if: env.NodeMajorVersion < 8 && env.NodeMajorVersion >= 6
+
+    # supertest for http calls
+    # - use 2.0.0 for Node.js < 4
+    # - use 3.4.2 for Node.js < 6
+    - name: Setup Node.js version-specific dependencies - supertest for http calls - use 2.0.0 for Node.js < 4
+      run: npm install --silent --save-dev supertest@2.0.0
+      if: env.NodeMajorVersion < 4
+      
+    - name: Setup Node.js version-specific dependencies - supertest for http calls  - use 3.4.2 for Node.js < 6
+      run: npm install --silent --save-dev supertest@3.4.2
+      if: env.NodeMajorVersion < 6 && env.NodeMajorVersion >= 4
+    
+        
+    # Update Node.js modules
+    # Prune and rebuild node_modules
+    - name: prune and rebuild node_modules
+      run: |
+        if (Test-Path -Path node_modules) {
+        npm prune
+        npm rebuild
+        }
+      shell: pwsh
```

Comment:
See my above comment, I don't think we should be using powershell in this config.

Reply:
My main reason for using PowerShell Core here was not having to duplicate steps when doing Linux vs Windows builds.  Using PowerShell Core, as this step above does, is what allows me to use this one step in both Windows and Linux (as well as Mac builds), and also allow me to specify the OS as part of the Matrix.

The other option is to have one job for running the Linux builds, with a Matrix of Node versions in it.  Those steps could be written in *nix'isms.  Then a second job for running the Windows builds, with a matrix of Node versions in it.  Those steps would be a duplicate of a lot of the steps in Linux, but where code is required, we would still have to use PowerShell Core.

Because of this, and because PowerShell Core runs on Linux/Windows/Mac, I thought take any steps that need to be cross platform and write them in PowerShell Core.

Reply:
I mean I don't even know what this does.  Am I wrong to think this can just be:

```
run: |
  npm prune
  npm rebuild
```

If there is no way to write this in a cross platform way, I would actually prefer to split it out to have a windows test run separate from the *nix ones.  Duplication is fine if it increases developer understanding, although I do agree if we can avoid it in a reasonable way that is obviously best.

Reply:
Ok, I hadn't looked at the travis and appveyor configs recently, but I do think if it means we can have one cross compatible script it is worth removing the check for the existence of `node_modules`.  Both of these are no-ops if there is no `node_modules` so we can just unwrap them from the if.

Reply:
@wesleytodd correct, we remove the if statement, then we can change that to what you have above.

Reply:
I believe that i added it because some of the older npms it was no a no op and they failed. Would need to validate they are no ops across the builds.

Reply:
I can remove it and run it and see what happens :)

Reply:
I've made this change.